src/engine/source/cmds/include/cmds/test.hpp

```diff
@@ -5,35 +5,34 @@
 #include <vector>
 
 #include <CLI/CLI.hpp>
+#include <base/utils/wazuhProtocol/wazuhProtocol.hpp>
+#include <cmds/apiclnt/client.hpp>
 
 namespace cmd::test
 {
 
-/**
- * @brief Run policy in test mode. Inputs from stdin, outputs event to stdout and
- * debug to stderr.
- *
- * @param kvdbPath Path to KVDB folder.
- * @param fileStorage Path to asset folders.
- * @param policy Name of the policy to be loaded.
- * @param logLevel Log level.
- * @param debugLevel Debug level.
- * @param assetTrace Trace specific assets.
- * @param protocolQueue Queue of the protocol.
- * @param protocolLocation Location of the protocol.
- */
+namespace details
+{
+constexpr auto ORIGIN_NAME {"engine_integrated_test_api"};
+} // namespace details
+
 struct Options
 {
-    std::string kvdbPath;
-    std::string fileStorage;
-    std::string policy;
-    std::string logLevel;
-    int debugLevel;
-    std::vector<std::string> assetTrace;
-    char protocolQueue;
+    std::string apiEndpoint;
+    std::string sessionName;
+    std::string event;
     std::string protocolLocation;
+    uint32_t protocolQueue;
+    int debugLevel;
 };
```

Comment:
We are mixing the "options" concept with "parameters" here

Reply:
It is the same case above, so they use it in the other commands and try to follow the same naming convention

Reply:
I find it confusing